tests/test_sota_checkpoints.py

```diff
@@ -466,46 +416,22 @@ def test_openvino_eval(self, openvino, sota_checkpoints_dir, sota_data_dir, ov_d
                          "{}/{config}.csv".format(config_folder, ov_data_dir, ir_model_folder,
                                                   PROJECT_ROOT, config=eval_test_struct.model_name_)
                 exit_code, err_str = self.run_cmd(ac_cmd, cwd=ACC_CHECK_DIR)
-
-        ac_metric = self.read_csv_metric(eval_test_struct.model_name_)
-        fp32_ac_metric = None
-        if eval_test_struct.reference_ is not None:
-            if os.path.exists('{}.csv'.format(eval_test_struct.reference_)):
-                fp32_ac_metric = self.read_csv_metric(eval_test_struct.reference_)
+                if exit_code != 0 or err_str is not None:
+                    assert False
```

Comment:
Use `pytest.fail` instead to fail with an explicit message.

Reply:
Done